### PR TITLE
avm report updates with some additional tweaks after font and variable changes

### PR DESCRIPTION
**Pull request by schooolman**
avm-report-update: style updates for the avm report block, as well as some updates to the icon card blocks that are included in the section.

- have added in the rest of the color variables found in the style guide.
- small updates to icon cards
- several updates to floating images after variables and font updates

Test URLs:
- Before: https://main--hsf-commonmoves--hlxsites.hlx.page/
- After: https://avm-report-update--hsf-commonmoves--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/avm-report/avm-report.css [hunk 1]
```diff
@@ -1,20 +1,29 @@
+main .avm-report-container.section {
+  background-color: var(--tertiary-color);
+  border-top: 1px solid var(--secondary-accent);
```
Comment (bstopp) [inline blocks/avm-report/avm-report.css:3]: What happens when the AVM report isn't the only thing in the section?

File: blocks/avm-report/avm-report.css [hunk 2]
```diff
@@ -71,4 +97,13 @@
     flex: unset;
     margin: 0;
   }
+
+  .avm-report-container .cards-wrapper {
```
Comment (bstopp) [inline blocks/avm-report/avm-report.css:101]: one block should not change the CSS of another

**Review by bstopp — CHANGES_REQUESTED** [review]

File: blocks/avm-report/avm-report.css (hunk 1 above)
Comment (schooolman) [reply blocks/avm-report/avm-report.css]: Have moved this new background color option to a new class in main styles file.

File: blocks/avm-report/avm-report.css (hunk 2 above)
Comment (schooolman) [reply blocks/avm-report/avm-report.css]: Have moved these styles into the cards block

**Review by bstopp — APPROVED** [review]

**Pull request merged**